src/sections/SideBar/components/NavLinks/SideBarLinks.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { makeStyles, Link, List, ListItem, ListItemIcon, ListItemText, createMuiTheme, ThemeProvider } from "@material-ui/core";
+import { HomeOutlinedIcon, PersonOutlineOutlinedIcon, MailOutlineOutlinedIcon, NotesOutlinedIcon } from "@material-ui/icons";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    maxWidth: 200,
+    backgroundColor: "rgb(61, 61, 61)",
+    color: "gray",
+    textTransform: "uppercase"
+  },
+  icon: {
+    color: "gray"
+  },
+  icon2: {
+    color: "white"
+  },
+  list: {
+    "& .Mui-selected": {
+      color: "white"
+    }
+  }
+}));
+
+const theme = createMuiTheme({
+  props: {
+    MuiButtonBase: {
+      disableRipple: true
+    }
+  }
+});
+
+const SelectedListItem = () => {
+  const classes = useStyles();
+  const [selectedIndex, setSelectedIndex] = React.useState(0);
+
+  const handleListItemClick = (event, index) => {
+    setSelectedIndex(index);
+  };
+
+  const selectIconColor = (index) => {
+    return (selectedIndex === index) ? classes.icon2 : classes.icon;
+  };
+
+  const arrValues = [["Home", <HomeOutlinedIcon className={selectIconColor(0)} />, "./home"], 
```

Comment:
the same generic component I mentioned before should help us here too, avoid setting up components inside a variable whenever is possible